src/containers/student-home-page/find-tutor-block/find-tutor-block.styles.js

```diff
@@ -1,7 +1,21 @@
 export const styles = {
+  container: {
+    mt: {
+      xs: '24px',
+      md: '48px'
+    },
```

Comment:
Delete these styles

Reply:
But we need margin to make some space, don't we ?

Reply:
As I see, you don't use it in code. So it is useless.

Reply:
in FindTutorBlock.jsx
<TitleBlock
      img={bag}
      style={styles.container}
      translationKey={translationKey}
    >